### ISSUE TITLE
TDLight branch

### ISSUE THREAD

**Issue by MrYacha** [aiogram/aiogram#470]
**Is your feature request related to a problem? Please describe.**
A specific branch for https://github.com/tdlight-team/tdlight-telegram-bot-api/ - fork of botapi
Its features - https://github.com/tdlight-team/tdlight-telegram-bot-api#tdlight-features

**Describe the solution you'd like**
A specific branch would be fine.

**Describe alternatives you've considered**
Not relevant

**Additional context**
None

**Comment by MasterGroosha** [comment]
Should TDLight branch have the same version as aiogram? e.g. "2.11.3" and "2.11.3-tdl"
OR
Should it have some different naming, so it can be updates asynchronously, e.g. "2.11.3" and "tdlight-1.0"?

**Comment by evgfilim1** [comment]
@MasterGroosha I think it'll be better to combine versioning like this: 2.11.3-tdl.1 < 2.11.3-tdl.1 (patch specific for tdlight) < 2.11.4-tdl.0 (new aiogram version)
But this approach doesn't really allow us to make breaking changes to tdlight only

**Comment by MasterGroosha** [comment]
@evgfilim1 can we move this discussion to "Discussions"?

**Comment by evgfilim1** [comment]
> @evgfilim1 can we move this discussion to "Discussions"?

If you have more to discuss, then why not

**Comment by JrooTJunior** [comment]
I dont think it should be different branch of aiogram. I'd like idea of creating an extension as separated package and different releases schedule.

Something like `aiogram-tdlight` package which contains TdlightBot class with new methods and types.

**Comment by andrew-ld** [comment]
https://github.com/tdlight-team/tdlight-telegram-bot-api/blob/gh-pages/tdlight-api-openapi.yaml

you can easily generate tdlight bot api objects using the open api schema

**Comment by JosXa** [comment]
Does this also concern the new user mode?

**Comment by andrew-ld** [comment]
yes
